### PR TITLE
add option to remove top panel

### PR DESCRIPTION
**Pull request by leb-kuchen**

### CODE REVIEW

**Comment by edfloreshz** [comment]
Isn't this the same as disabling the top panel in COSMIC Settings?

**Comment by git-f0x** [comment]
I believe only the Dock can be disabled. There isn't a toggle for that in Panel settings.

**Comment by edfloreshz** [comment]
True! I'll review this a bit later then.

File: src/pages/panel.rs [hunk 1]
```diff
@@ -50,6 +91,10 @@ impl Panel {
 
         widget::scrollable(
             widget::settings::view_section("Panel")
+                .add(
+                    widget::settings::item::builder("Show panel")
```
Comment (edfloreshz) [inline src/pages/panel.rs:95]: Could you add a key for this in `i18n/en/cosmic_tweaks.ftl`?

**Review by edfloreshz — CHANGES_REQUESTED** [review]
Please resolve the comment and we can merge 👍🏼

**Review by edfloreshz — APPROVED** [review]
LGTM

**Comment by edfloreshz** [comment]
Thanks! 🎉

**Pull request merged**